BraveRewardsUI/RewardsPanelController.swift

```diff
@@ -24,7 +24,11 @@ public class RewardsPanelController: PopoverNavigationController {
   public override func viewDidLoad() {
     super.viewDidLoad()
     
+    let defaultLightNavColor = UIColor(red: (247/255), green: (247/255), blue: (247/255), alpha: 1)
```

Comment:
Where does this come from?

Reply:
From stack overflow :P, the goal is to use default color for navigation bar

Reply:
Is this only needed for iOS 12 then?

Reply:
youre right, fixed it